content/applications/sales/crm/track_leads/prospect_visits.rst

```diff
@@ -1,55 +1,140 @@
-===========================
-Track your prospects visits
-===========================
+====================
+Track webpage visits
+====================
 
-Tracking your website pages will give you much more information about
-the interests of your website visitors.
+Tracking the number of visits to individual webpages provides valuable insight about the interests
+of website visitors. It can also be used to analyze the success of marketing campaigns.
 
-Every tracked page they visit will be recorded on your lead/opportunity
-if they use the contact form on your website.
+With Odoo, tracking links can be created to track visits for any page on the website. When a lead is
+created from a website contact form, every page that the prospective customer visits is recorded on
+the lead/opportunity form in the *CRM* app.
 
-Configuration
-=============
+.. _crm/track_leads/link-tracker:
 
-To use this feature, install the free module *Lead Scoring* under your
-*Apps* page (only available in Odoo Enterprise).
+Track a webpage
+===============
+
+Odoo's *link tracker* creates a unique tracking link for any page on a company website. The link
+contains analytics trackers that can be used in Google Analytics to track clicks and visitors, as
+well as in Odoo Reports to track opportunities and revenue.
+
+The link can also be sent as part of a marketing campaign to track clicks and webpage views for that
+particular campaign.
 
-.. image:: prospect_visits/prospect_visits01.png
+To set up tracking for a webpage, go to any static page on the website through the *Website* app.
+Then, click on the :guilabel:`Site` header menu at the top of the page, and select :guilabel:`Link
+Tracker`.
```

Comment:
Since we are directing the user to the _Website_ app, please use a :menuselection:
```suggestion
To set up tracking for a webpage, go to any static page on the website through the
:menuselection:`Website` app. Then, click on the :guilabel:`Site` header menu at the top of the
page, and select :guilabel:`Link Tracker`.
```